app/controllers/endpoints_controller.rb

```diff
@@ -21,6 +21,11 @@ def create
     endpoint = Endpoint.new(endpoint_params)
     endpoint.account = current_user.account
     endpoint.type ||= Endpoint.name
+
+    nested_filters.each do |filter_params|
```

Comment:
`filter_params` is a two element array (like `[:app_ids, [1, 2, 3]]`).

Changing this to:
```ruby
authorize endpoint.filters.build(nested_filters.merge(account: current_user.account))
```
seems to work for `app_ids` and `event_type_ids`, but fails with `ActiveRecord::AssociationTypeMismatch: SeverityFilter(#70042269683180) expected, got "low" which is an instance of String(#47374285323740)` for severity filters